src/clients/mobiles/remote/index.tsx

```diff
@@ -104,45 +110,48 @@ class RemoteContainer extends React.Component<RemoteProps & ReduxStore, {}>  {
             volume: 30,
             volumeAsPercentage: '30%'
         }
-        return (<div className={styles.container}>
+        return (<div className={classnames(styles.container, 
+                (isRGBForWhite.apply(null, rgbToObject(current.dominantBackgroundColor)) ? styles.dark : styles.white))}>
```

Comment:
Wanted to use spread operator here but it outputted errors :/ 
If we could look at this @wdelmas

Reply:
ok, we'll see together.